### PR TITLE
chore: update next workflows

### PR DESCRIPTION
**Pull request by andyesp**

### CODE REVIEW

File: .github/workflows/next_pull_request_build.yml [hunk 1]
```diff
@@ -9,41 +9,10 @@ on:
...
-
-      - name: building
-        run: npm run build
+  build:
+    uses: decentraland/actions/.github/workflows/build-quay-main.yml@main
+    with:
+      service-name: governance
+    secrets:
+      QUAY_USERNAME: ${{ secrets.QUAY_USERNAME }}
+      QUAY_TOKEN: ${{ secrets.QUAY_TOKEN }}
```
Comment (menduz) [inline .github/workflows/next_pull_request_build.yml:18]: this whole file shouldn't be necessary anymore I think

File: .github/workflows/next_build.yml [hunk 2]
```diff
@@ -1,46 +1,15 @@
...
-
-      - name: building
-        run: npm run build
+  build:
+    uses: decentraland/actions/.github/workflows/build-quay-main.yml@main
+    with:
+      service-name: governance
+    secrets:
+      QUAY_USERNAME: ${{ secrets.QUAY_USERNAME }}
+      QUAY_TOKEN: ${{ secrets.QUAY_TOKEN }}
```
Comment (menduz) [inline .github/workflows/next_build.yml:15]: I think you are forgetting to deploy the image to a specific env https://github.com/decentraland/sdk-content-server/blob/main/.github/workflows/docker.yml#L13

Comment (andyesp) [reply .github/workflows/next_build.yml:15]: Done! So if we want to deploy to all 3 environments we need to create a new workflow for each one. Right?

File: .github/workflows/next_pull_request_build.yml (hunk 1 above)
Comment (andyesp) [reply .github/workflows/next_pull_request_build.yml:18]: Removed. It'd be nice to have deployments per pull request but we can review that later after the deploy on merge is working ok as it probably needs a different setup.

File: .github/workflows/next_build.yml (hunk 2 above)
Comment (menduz) [reply .github/workflows/next_build.yml:15]: separate them by space `dev stg prd`

**Review by ncomerci — APPROVED** [review]

**Pull request merged**